### PR TITLE
WIP: update of urllib2 usage to Python 3

### PR DESCRIPTION
**Pull request by neteler**
Please review, not sure if this change is sufficient

### CODE REVIEW

File: grass7/raster/r.in.srtm.region/r.in.srtm.region.py [hunk 1]
```diff
@@ -93,10 +93,16 @@
 
 import os
 import atexit
-import urllib
-import urllib2
+try:
+    from urllib2 import urlopen, URLError, HTTPError
```
Comment (petrasovaa) [inline grass7/raster/r.in.srtm.region/r.in.srtm.region.py]: This should replace all the urllib stuff:

    from six.moves.urllib import request as urllib2

Comment (petrasovaa) [reply grass7/raster/r.in.srtm.region/r.in.srtm.region.py]: Line

    urllib.urlcleanup()

should be changed to 

    urllib2.urlcleanup()

(with the import above)

**Comment by neteler** [comment]
Thanks for your suggestions! I have change it accordingly but it seems to hang when trying to establish the connection to https://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1.003/2000.02.11/ using the [r.in.srtm.region manual page](https://grass.osgeo.org/grass7/manuals/addons/r.in.srtm.region.html) example:

```
strace -p 10771
strace: Process 10771 attached
connect(3, {sa_family=AF_INET6, sin6_port=htons(443), sin6_flowinfo=htonl(0), inet_pton(AF_INET6, "2001:49c8:4000:127d::133:130", &sin6_addr), sin6_scope_id=0}, 28
```
Something is still missing...

**Comment by petrasovaa** [comment]
Weird, I tested the manual example and it worked with Python 3.5

**Comment by neteler** [comment]
I must have used an outdated version in the PATH. Works fine also for me.
Thanks for your support, @petrasovaa

**Pull request merged**